__tests__/writingFile.test.js

```diff
@@ -0,0 +1,22 @@
+const data = require('../lib/data');
```

Comment:
Normally it's better to keep the test file in the same directory as the tested module to make it easier to find one another. Move this file.

Reply:
Moved :)